src/screens/Menu.js

```diff
@@ -206,6 +206,13 @@ const MenuPage = () => (
           <PropertyValue type="array[object]">
             <Example> {`[{}, {}]`} </Example>
           </PropertyValue>
+          <PropertyValue type="array[array[object]]">
+            <Description>
+              Items can be defined as an array of arrays of objects to create
+              groups of Menu items.
+            </Description>
+            <Example> {`[[{}], [{}]]`} </Example>
```

Comment:
Would it be more valuable to include more detail about what that object might look like? (And as a bonus, improve the non-group menu example as well)

Reply:
Good idea, I added this in the latest commit